src/BackgroundImage.js

```diff
@@ -1,14 +1,10 @@
 import Box from './Box'
-import { style, responsiveStyle } from 'styled-system'
+import { height, ratio, style, responsiveStyle } from 'styled-system'
 
 const image = props => props.src ? ({
   backgroundImage: `url(${props.src})`
 }) : null
 
-const height = responsiveStyle({
-  prop: 'height'
-})
-
 const position = style({
```

Comment:
styled-system 2 includes a `backgroundPosition` function, but the prop name is `backgroundPosition`